compiler/visq/visqlib/QErrorComputer.py

```diff
@@ -198,3 +199,46 @@ def get_final_result(self):
     def run(self):
         self.advance_on(self._fp32_dir, self._fq_dir)
         return self.get_final_result()
+
+
+# Scaled Root Mean Square Error (SRMSE)
+# SRMSE = sqrt(MSE) / scale
+class SRMSEComputer(QErrorComputer):
+    def __init__(self, fp32_dir, fq_dir):
+        super().__init__(fp32_dir, fq_dir)
+        self.scale_file = Path(fq_dir) / 'scales.txt'
+
+    # Incrementally compute Qerror while traversing all data in fp32_dir and fq_dir
+    def advance_on(self, fp32_dir, fq_dir):
+        data_paths = self.collect_data_path(fp32_dir, fq_dir)
+
+        for tensor_name, data_path in data_paths.items():
+            for (fp32_data_path, fq_data_path) in data_path:
+                fp32_data = np.load(fp32_data_path)
+                fq_data = np.load(fq_data_path)
+
+                MSE = np.square(fp32_data - fq_data).mean()
+
+                if tensor_name in self.qerror_map:
+                    self.qerror_map[tensor_name] += MSE
+                else:
+                    self.qerror_map[tensor_name] = MSE
```

Comment:
If `qerror_map` only contains float value. It would be helpful.

```suggestion
qerror_map = defaultdict(float)

self.qerror_map[tensor_name] += MSE

```

Reply:
Thanks! It looks much more concise.

Reply:
There are other codes using `dict`. I'll handle this comment in another PR.